src/Elmah.AzureTableStorage/AzureTableStorageErrorLog.cs

```diff
@@ -96,7 +93,8 @@ public override string Log(Error error)
                 StatusCode = error.StatusCode,
                 Type = error.Type,
                 User = error.User,
-            };
+            };
+            elmahEntity.SetXml(ErrorXml.EncodeString(error));
```

Comment:
I think this feature should be optional, maybe via appSettings. If people are retrieving their Elmah data in a different way, like querying directly, they wouldn't want the GZip functionality and, to them, this would be a breaking change.